src/pymmcore_widgets/_load_system_cfg_widget.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+
+from qtpy import QtWidgets as QtW
+
+from ._core import load_system_config
+
+
+class ConfigurationWidget(QtW.QGroupBox):
+    """Widget to select and load MM configuration."""
+
+    def __init__(self, parent: Optional[QtW.QWidget] = None) -> None:
```

Comment:
we should be allowing all of these widgets to take in a (non-global) core object.  so `load_system_configuration` should be modified to take an optional `mmcore` argument, and this widget should also take one in the init

Reply:
@tlambert03 fixed